src/WorkOS.net/WorkOS.net.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;net7.0;netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Could we split the changes to the compilation targets out into a separate PR?

Reply:
Moved to https://github.com/workos/workos-dotnet/pull/134